llvm/test/CodeGen/Mips/inlineasm-constraints-softfloat.ll

```diff
@@ -0,0 +1,25 @@
+; RUN: llc  -march=mips < %s | FileCheck %s --check-prefix=MIPS32
+; RUN: llc  -march=mips64 < %s | FileCheck %s --check-prefix=MIPS64
+
+define dso_local void @read_double(double* nocapture noundef readonly %0) local_unnamed_addr #0 {
```

Comment:
Newer tests should use opaque pointers (`ptr`)

Reply:
Thanks.